.github/workflows/build-java.yaml

```diff
@@ -36,6 +36,9 @@ jobs:
         uses: Swatinem/rust-cache@v2
       - name: Build yggdrasil
```

Comment:
Could we take this one out? We don't need this anymore and it just adds to the build time